packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -161,7 +207,7 @@ const CancelSubscriptionPanel = ({
                   data-testid="confirm-cancel-subscription-checkbox"
                   type="checkbox"
                   defaultChecked={confirmationChecked}
-                  onChange={onConfirmationChanged}
+                  onChange={engagedOnConfirmation}
```

Comment:
This is also a loss of readability (`engagedOnConfirmation` vs `onConfirmationChanged`), same basic feedback as line 221 below.